### PR TITLE
Convert thirtyinch module tests to Kotlin

### PR DESCRIPTION
**Pull request by lukaszkalnik**
Adds Kotlin support to `thirtyinch` module tests. 
As a start, converts `TiTestPresenterTest` and `TestPresenterExampleTest` to Kotlin and MockK.

### CODE REVIEW

File: thirtyinch/build.gradle [hunk 1]
```diff
@@ -37,11 +38,13 @@ configurations {
 
 dependencies {
     compileOnly "com.android.support:appcompat-v7:$supportLibraryVersion"
```
[the remark this answers is not in the document]
Comment (lukaszkalnik) [reply thirtyinch/build.gradle]: Thank you.

**Comment by StefMa** [comment]
You have to update the travis.yml file too

**Comment by lukaszkalnik** [comment]
Ok, I think we can do it in a separate PR.

**Comment by lukaszkalnik** [comment]
As discussed in a conversation with @passsy the library is pretty stable right now and at this moment there is no need to convert the source code (especially tests) to Kotlin.
Instead of reviewing this and related PRs we can direct our efforts to more valuable goals.
So I'm closing it and all related PRs for now

**Pull request closed without merging**